docs/package.json

```diff
@@ -9,7 +9,7 @@
     "build:clean": "rimraf .next && pnpm build",
     "build-sw": "node ./scripts/buildServiceWorker.js",
     "dev": "next dev --port 3001",
-    "deploy": "git push -f upstream master:docs-v7",
+    "deploy": "git push -f upstream master:docs-next",
```

Comment:
If we change the nomenclature of the branch we work on, should we also do the same for the doc branches?
(use `docs-v8` instead of `docs-next`)?

Reply:
But we are not using `v7.mui.com` and yet we are using `docs-v7` and not `docs` :innocent: 

You are right, it's not worth the headache anyway

Reply:
Also outside of this PR, but did we consider using v8.mui.com instead of next.mui.com ?

It would avoid ending with a next.mui.com that is not the next version anymore after major being stable

Reply:
I think we have brought this topic up, but I believe the problem would be similar—what do we do with `v8` when it becomes the new current release? 🤔

Reply:
@alexfauquette if we do that then I think `v7.mui.com` should redirect to `mui.com` so that all the currently available major version (stable or not) have their sub-domain.

No strong preference.
I don't know how commonly used the `next.xxx.yy` pattern is.

Reply:
@LukasTy I guess the advantage is that we only have one sub domain that see it's content jump from major to major (the main one, `mui.com`, not sure if you call that a sub domain though :laughing: )

`v8.mui.com` always target the v8 major, when v8 is in alpha it shows the alpha doc, when v8 is stable it redirects to `mui.com` (or at least it should to avoid having to update `docs-v8` all the time during the stable), and when we start the v9 alpha we sync `docs-v8` and people can then use `v8.mui.com` to see the old v8 doc.

Reply:
we could basically do both as well ... `next.mui.com` can also always target the next major release.

> I don't know how commonly used the next.xxx.yy pattern is.

@flaviendelangle I think most libraries do not even use subdomains tbh ... not sure if this is a very commonly used practice

Side benefit: The `getVersionOptions` function would get easier: e.g.:
```tsx
getVersionOptions('introduction', [8, process.env.LIB_VERSION, 6, 5])

// definition
const getVersionOptions = (id, versions) =>
  versions.map((version) => {
    if (version === process.env.LIB_VERSION) {
      return {
        current: true,
        text: version,
        href: `${languagePrefix}${productIdSubpathMap[id]}/`,
      };
    }
    return {
      text: version,
      href: `https://v${version}.mui.com${languagePrefix}${productIdSubpathMap[id]}/`,
    };
  });
```

Reply:
> Also outside of this PR, but did we consider using v8.mui.com instead of next.mui.com ?

@alexfauquette we should discuss this during the next weekly. TBH it makes as much sense as the `next` approach, but would induce a lot less friction during the transition. 

I think the main reason we had this is to align it with the `next` npm tag.

After doing this I think we can also introduce an automation for the major prep ... I'll take note on that! :D

Reply:
I'm all for simplicity. 👍 
I think I've seen the vX.domain pattern used, it's not an edge-case. 👍 

What is @romgrk take on it?
Given that you suggested foregoing the `next` branch "tango". 😆

Reply:
I like `next.mui.com` because it tells the user right away that this version isn't yet stable/usable, but if that information is transmitted otherwise (e.g. a banner in the header), I'm fine either way.